services/datamanager/datasync/sync.go

```diff
@@ -1,368 +0,0 @@
-// Package datasync contains interfaces for syncing data from robots to the app.viam.com cloud.
```

Comment:
I removed the datasync package as the only thing that used it was the builtin data manager, however the package higherarchy made it seem like it was (from a package dependency perspective) a sibling of the datamanager.datacapture package and able to be used independently of the builtin data manger, which is false.